frontend/src/pages/GranteeSearch/components/GranteeResults.js

```diff
@@ -22,15 +23,17 @@ export default function GranteeResults(
   const getClassNamesFor = (name) => (sortConfig.sortBy === name ? sortConfig.direction : '');
 
   const renderGrantee = (grantee) => {
+    // Get a unique sorted list of Program Specialists.
+    const psList = getDistinctSortedArray(grantee.grants, 'programSpecialistName').join(', ');
```

Comment:
I was under the impression based on the standup discussion that these should be separated by new lines not commas

Reply:
I believe newline vs comma was centered around the CSV. I would _think_ using a comma here is fine.